crates/rattler_conda_types/src/conda_lock/mod.rs

```diff
@@ -239,14 +294,60 @@ mod test {
         )
     }
 
+    fn lock_file_path_python() -> String {
+        format!(
+            "{}/{}",
+            env!("CARGO_MANIFEST_DIR"),
+            "../../test-data/conda-lock/python-conda-lock.yml"
+        )
+    }
+
     #[test]
     fn read_conda_lock() {
         // Try to read conda_lock
-        let conda_lock: CondaLock =
-            from_str(&std::fs::read_to_string(lock_file_path()).unwrap()).unwrap();
+        let conda_lock = CondaLock::from_path(Path::new(&lock_file_path())).unwrap();
         // Make sure that we have parsed some packages
         insta::with_settings!({sort_maps => true}, {
         insta::assert_yaml_snapshot!(conda_lock);
-        })
+        });
+    }
+
+    #[test]
+    fn read_conda_lock_python() {
+        // Try to read conda_lock
+        let conda_lock = CondaLock::from_path(Path::new(&lock_file_path_python())).unwrap();
+        // Make sure that we have parsed some packages
+        insta::with_settings!({sort_maps => true}, {
+        insta::assert_yaml_snapshot!(conda_lock);
+        });
+    }
+
+    #[test]
+    fn packages_for_platform() {
+        // Try to read conda_lock
+        let conda_lock = CondaLock::from_path(Path::new(&lock_file_path())).unwrap();
+        // Make sure that we have parsed some packages
+        assert!(!conda_lock.package.is_empty());
+        assert_eq!(
+            conda_lock
+                .packages_for_platform(Platform::Linux64)
+                .collect::<Vec<_>>()
+                .len(),
+            254
+        );
+        assert_eq!(
+            conda_lock
+                .packages_for_platform(Platform::Osx64)
+                .collect::<Vec<_>>()
+                .len(),
+            180
+        );
+        assert_eq!(
+            conda_lock
+                .packages_for_platform(Platform::OsxArm64)
+                .collect::<Vec<_>>()
+                .len(),
+            183
+        );
```

Comment:
This is a bit fragile. You could also use `insta` to print all the filenames of the packages.

Reply:
This is done.